files/justfiles/utilities.just

```diff
@@ -184,16 +184,14 @@ install-steam:
     echo "Please select a method to install steam:"
     echo "    1) Flatpak - will install the steam flatpak from flathub-unverified"
     echo "    2) Distrobox - will set up steam via the bazzite-arch distrobox image"
-    echo "    3) Native layer - will install the steam rpm from negativo17"
     while [[ "$valid_input" == "0" ]]; do
-        read -p "Selection [1-3]: " method_selection
-        if [[ "$method_selection" == [123]* ]]; then
+        read -p "Selection [1-2]: " method_selection
+        if [[ "$method_selection" == [12]* ]]; then
             valid_input="1"
         else
             echo "That is not a valid selection."
         fi
     done
-    valid_input="0"
```

Comment:
@spaceoden fyi i removed this as well since it doesn't have any effect

Reply:
oops. good catch.